src/Tribe/RSVP/Service_Provider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tribe\Tickets\RSVP;
+
+/**
+ * Class Service_Provider
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\RSVP
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->register_early_access();
+	}
+
+	/**
+	 * Registers the RSVP Early Access Service.
+	 *
+	 * @since TBD
+	 */
+	private function register_early_access() {
+		add_action( 'init', [
+			$this->container->make( Early_Access::class ),
+			'maybe_register_early_access_assets'
+		]);
+
+		add_action( 'wp_enqueue_scripts', [
+			$this->container->make( Early_Access::class ),
+			'maybe_deregister_rsvp_assets'
+		], 100 );
+
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 387d3552f0463074cbf1527bb43b7139</sub>